modules/blocks.php

```diff
@@ -60,6 +66,37 @@ function jetpack_tiled_gallery_load_block_assets( $attr, $content ) {
 	}
 }
 
+/**
+ * Giphy block registration/dependency declaration.
+ *
+ * @param array $attr - Array containing the map block attributes.
+ *
+ * @return string
+ */
+function jetpack_giphy_block_load_assets( $attr ) {
+	$align       = isset( $attr['align'] ) ? $attr['align'] : 'center';
+	$style       = 'padding-top:' . $attr['paddingTop'];
+	$giphy_url   = isset( $attr['giphyUrl'] ) ? $attr['giphyUrl'] : '//giphy.com/embed/ZgTR3UQ9XAWDvqy9jv';
```

Comment:
Shouldn't we abort rendering anything if we don't have a URL? It seems unlikely users will insert the block without searching and want the placeholder gif.

Reply:
Not sure I would agree. Seems like the view-side should always be a representation of what the user sees in the editor, and if they drag in the block but perform no search, they'll see the placeholder. Hopefully in this case, seeing the placeholder in the Preview is a loud reminder that they should use the block to search for the content they want, but I think that suppressing the block entirely ends up being more confusing overall.

Reply:
> Seems like the view-side should always be a representation of what the user sees in the editor, and if they drag in the block but perform no search, they'll see the placeholder.

I agree with this premise. Which leads me to believe that we should use a true Gutenberg placeholder as the placeholder in the editor.

I wasn't involved in that part of the discussion, but I would like to raise the concern here before moving on. We can change this pretty simply in a follow-up.

Should we show the editor "placeholder" gif on the frontend if the user hasn't changed the gif? Should we use a standard Gutenberg placeholder?

cc: @thomasguillot

Reply:
Resolved in https://github.com/Automattic/wp-calypso/pull/30402 and https://github.com/Automattic/jetpack/pull/10989/commits/bd0a2818a78154c0ef7e71e30cc7fbd15d91f43c